.github/workflows/release.yml

```diff
@@ -71,24 +71,13 @@ jobs:
   verification:
     needs: [goreleaser, provenance]
     runs-on: ubuntu-latest
+    env:
+      ATT_FILE_NAME: "attestation.intoto.jsonl"
```

Comment:
is this `attestation.intoto` or `multiple`?

Reply:
I'm not sure if multiples are fully supported yet but it works for me with the latest version [here](https://github.com/mchmarny/vulctl/blob/07e5cf2966f7629af3373bda0d3d8238dcd0c6f5/.github/workflows/on-tag-cli.yaml#L108-L114). In that case, the attestation file generated by SLSA Generator using `goreleaser` was `multiple.intoto.jsonl`

Reply:
here is the PR that fixes this problem: https://github.com/ko-build/ko/pull/983/files

you may rebase from main